libs/shared/src/entities/message-template/channel.enum.ts

```diff
@@ -2,6 +2,7 @@ export enum ChannelTypeEnum {
   IN_APP = 'in_app',
   EMAIL = 'email',
   SMS = 'sms',
+  DIGEST_BACKOFF = 'digest_backoff',
```

Comment:
@davidsoderberg  A thought, maybe it's worth calling this `DIGEST` rather than strategy of execution + digest. For those cases we will have a step of type digest and we will have more metadata associated to this step such as "strategy" more info like time window etc.

In this case we can associate a step to a digest type and then use other metadata for configuration sake.